chaincfg/params.go

```diff
@@ -176,18 +174,15 @@ var RegressionNetParams = Params{
 	// Checkpoints ordered from oldest to newest.
 	Checkpoints: nil,
 
-	// Reject version 1 blocks once a majority of the network has upgraded.
-	// 75% (75 / 100)
-	// This is part of BIP0034.
-	BlockV1RejectNumRequired: 75,
-	BlockV1RejectNumToCheck:  100,
-
-	// Ensure coinbase starts with serialized block heights for version 2
-	// blocks or newer once a majority of the network has upgraded.
+	// Enforce current block version once majority of the network has
+	// upgraded.
 	// 51% (51 / 100)
-	// This is part of BIP0034.
-	CoinbaseBlockHeightNumRequired: 51,
-	CoinbaseBlockHeightNumToCheck:  100,
+	// Reject previous block versions once a majority of the network has
+	// upgraded.
+	// 75% (75 / 100)
+	BlockEnforceNumRequired: 51,
```

Comment:
Since the regression network is shared with BC, these should be updated to match.